### ISSUE TITLE
ironmq CI tests fail

### ISSUE THREAD

**Issue by pjfanning** [apache/pekko-connectors#697]
In all recent PR builds.
```
Creating network "pekko-connectors_default" with the default driver
Creating volume "pekko-connectors_kudu-tserver-data" with default driver
Pulling ironauth (iron/auth:)...
latest: Pulling from iron/auth
[DEPRECATION NOTICE] Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of docker.io/iron/auth:latest to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2. More information at https://docs.docker.com/go/deprecated-image-specs/
Error: Process completed with exit code 1.
```

**Comment by laglangyue** [comment]
refference to
https://github.com/etcd-io/etcd/discussions/17811
maybe we shoulde add a version for iron/auth and iron/mq
![image](https://github.com/apache/pekko-connectors/assets/35491928/86a90534-e771-46cf-a4cd-63ff05ca1acc)

**Comment by pjfanning** [comment]
https://hub.docker.com/r/iron/mq is unmaintained. No images have been published in years.

**Comment by samueleresca** [comment]
> https://hub.docker.com/r/iron/mq is unmaintained. No images have been published in years.
Yes. I think the error won't go away unless there is an image compliant with OCI standard / Docker v2. 

We either:
- Don't test on iron/mq
- Republish a ironmq image that is compliant with OCI standard.
- Reach out the ironmq team

**Comment by pjfanning** [comment]
I emaile IronMQ support earlier. I'll see if they respond.

**Comment by Roiocam** [comment]
> refference to [etcd-io/etcd#17811](https://github.com/etcd-io/etcd/discussions/17811) maybe we shoulde add a version for iron/auth and iron/mq ![image](https://private-user-images.githubusercontent.com/35491928/340071700-86a90534-e771-46cf-a4cd-63ff05ca1acc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg1MzA4OTksIm5iZiI6MTcxODUzMDU5OSwicGF0aCI6Ii8zNTQ5MTkyOC8zNDAwNzE3MDAtODZhOTA1MzQtZTc3MS00NmNmLWE0Y2QtNjNmZjA1Y2ExYWNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE2VDA5MzYzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmYjVkN2UxYTE1MzllMWUzZjQ5YTJjOGE1MmQ5ZjFkNjk2M2QzYmI2MDM1YmU1ZDYzOTAzYzczZjkyZjI3OGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dHGBEfIVh5ONZLjqqQsYdqnAvHE6gdiOR5SO__Q-hYo)

I think that's not quiet the same thing:

- the refference issue is because etcd team didn't use v2 format to publish their latest tag.
- this issue is because the docker image is not even maintained in docker hub...

**Comment by Roiocam** [comment]
> I emaile IronMQ support earlier. I'll see if they respond.

Thanks. 

In addition, i think ironmq team required users to build the docker image from themself, see: https://github.com/iron-io/dockerworker. Do you think this information will be helpful?

**Comment by pjfanning** [comment]
The code in https://github.com/iron-io/dockerworker is 8/9 years old. Has checked in jars.
We really have to ask ourselves, who would be using this sort of unmaintained stuff and whether Pekko should support it.

**Comment by laglangyue** [comment]
I `git pull` and `git tag` the image, the manifest will changed to schema-v2.

We have some ways to solve it:
1. drop this module
2. git pull, git tag, git push to pekko in dockerhub,  This may involve some license issues
3. Waiting for iron to fix it, but I don't think they will because the Dockerhub image has not been maintained for a long time
so should we 
4. just disable the test


I aggre to drop the module.

**Comment by raboof** [comment]
I agree dropping the module seems reasonable. If someone steps up and offers to revive/maintain the module they can do so outside of the ASF.

Might be worth highlighting on the dev list?

**Comment by pjfanning** [comment]
IronMQ support suggested trying this env variable.
DOCKER_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE=true

This does not help on my Mac laptop. I found the setting in Docker docs but Googling it doesn't return many matches. Even if it worked, it only buys us a bit of time until Docker drops support for the old image format altogether.
